### DISCUSSION TITLE
How to convert between S2geo type value and WKB

### DISCUSSION THREAD

**Discussion by LemonLiTree** [google/s2geometry#309] [Q&A]
hi,everyone
I want to develop two functions,
1. Convert S2geo type to WKB representation.
2.Converts an expression for a hexadecimal-text STRING or BYTES value (WKB) into a S2geo type value.
Is there any information available for reference？
thanks

**Comment by smcallis** [comment]
What's the S2geo type?  I'm not familiar with that.

**Reply by LemonLiTree** [reply]
S2Point / s2polygon/s2polyline etc
